SwiftWisdom/Core/StandardLibrary/Collection/Collection+Random.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Collection+Random.swift
+//  SwiftWisdom
+//
+//  Created by Andrew Dolce on 2/17/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+public extension Collection {
+    public func ip_random() -> Self.Iterator.Element {
+        return ip_random(withUniformRNGFunction: arc4random_uniform)
```

Comment:
I can remove this. The only purpose of it was to let the tests specify a mock RNG function.